rides-android/src/main/java/com/uber/sdk/android/rides/RideRequestButton.java

```diff
@@ -225,23 +225,38 @@ private void setTextAttributes(
                 attrsResources,
                 defStyleAttr,
                 defStyleRes);
+
+        TypedArray textSizeAttributes = context.getTheme().obtainStyledAttributes(
+                attrs,
+                R.styleable.RideRequestButton,
+                defStyleAttr,
+                defStyleRes);
+        int primaryTextSize = (int) textSizeAttributes.getDimension(
+                R.styleable.RideRequestButton_ub__primary_text_size,
+                getResources().getDimension(R.dimen.ub__text_size));
+        int secondaryTextSize = (int) textSizeAttributes.getDimension(
+                R.styleable.RideRequestButton_ub__secondary_text_size,
+                getResources().getDimension(R.dimen.ub__secondary_text_size));
+        textSizeAttributes.recycle();
+
         try {
-            applyTextAttributes(priceEstimateView, textAttributes);
-            applyTextAttributes(timeEstimateView, textAttributes);
-            applyTextAttributes(requestButton, textAttributes);
+            applyTextAttributes(priceEstimateView, textAttributes, secondaryTextSize);
+            applyTextAttributes(timeEstimateView, textAttributes, secondaryTextSize);
+            applyTextAttributes(requestButton, textAttributes, primaryTextSize);
         } finally {
             textAttributes.recycle();
         }
     }
 
     @SuppressLint("ResourceType")
-    private void applyTextAttributes(TextView textView, TypedArray textAttributes) {
+    private void applyTextAttributes(TextView textView, TypedArray textAttributes, int textSize) {
```

Comment:
Since we're already pulling in textSize as an attribute, shouldn't we just be able to reference getInt(2) and call setTextSize() without needing new values?